spec/spec.md

```diff
@@ -638,7 +638,325 @@ Below is a storyboard that explains one concrete scenario using a workplace cred
 
 ## Examples
 
-Embedded or referenced non-normative examples
+The examples are designed to enable you to test portions of the protocol flow with your own implementations.
+
```

Comment:
The examples are so good I think non-normative needs to go back in `The examples which follow are non-normative.`

would suggest adding something like 
"Each example is consistent with the other examples in the section,  but will not match any externally run system and should not be considered as supported or endorsed identity. " to indicate that the examples hold together and even though they look real they should not be considered anything more than examples.

Reply:
I like this. more eloquent than what i was trying.

Reply:
I am not sure what "supported or endorsed identity" means

Reply:
i wanted to portray that the identifiers are for example purposes only and shouldnt be used for anything real.